site/installing.md

```diff
@@ -4,42 +4,51 @@ menu_order: 20
 search_type: Documentation
 ---
 
+Weave Scope consists of three parts: the probe, the app and the user interface.  Scope can be deployed in either a standalone configuration, where you run everything yourself, or you can use Weave Cloud, in which case only the probes run in your environment, and the app and UI are hosted by Weave Cloud.
 
-Weave Scope consists of three parts: the probe, the app and the user interface.  Scope can be deployed in either a standalone configuration, where you run everything yourself, or you can use Weave Cloud.
+ * [Installing on any Platform and Orchestrator, via Weave Cloud](#weave-cloud)
+ * [Installing on Docker](#docker)
+   * [Single-node](#docker-single-node)
+   * [Cluster](#docker-cluster)
+   * [Cluster with Weave Net](#docker-cluster-net)
+   * [Using Docker Compose](#docker-compose)
+ * [Installing on Orchestrators](#orchestrators)
+   * [Kubernetes](#k8s)
+   * [OpenShift](#ose)
+   * [Amazon ECS](#ecs)
+   * [minimesos](#minimesos)
+   * [Mesosphere DC/OS](#dcos)
+
+## <a name="weave-cloud"></a>Installing on any Platform and Orchestrator, via Weave Cloud
 
-Weave Cloud is the recommended option if:
+Weave Cloud is a SaaS that simplifies deployment, monitoring and
+management of your containers and microservices. Installing Weave
+Scope via Weave Cloud is the recommended option if:
 
+ * You are new to Weave Scope.
  * You are deploying to larger clusters.
  * You require secure remote access.
  * You want to share access with your coworkers.
  * You want to minimize Weave Scope memory and CPU usage.
+ * You want to benefit from Weave Cloud features beyond Weave Scope.
 
-The following topics are discussed:
+Weave Cloud provides easy step-by-step instructions for installation
+on a variety of platforms and orchestrators. To get started with Weave
+Cloud, [sign up for a free trial](https://cloud.weave.works/).
 
```

Comment:
Add a link to the install pages in the Weave Cloud docs here.

Reply:
The installation instructions are embedded in Weave Cloud. Also, we wouldn't want users to wander off to reading some install docs here.